numba/unicode.py

```diff
@@ -1143,6 +1144,91 @@ def impl(a):
     return impl
 
 
+@overload_method(types.UnicodeType, 'islower')
+def unicode_islower(a):
+    """
+    Implements .islower()
+    """
+    # impl is an approximate translation of:
+    # https://github.com/python/cpython/blob/1d4b6ba19466aba0eb91c4ba01ba509acf18c723/Objects/unicodeobject.c#L11794-L11827
+    def impl(a):
+        l = len(a)
+        if l == 0:
+            return False
+        if a._is_ascii:
+            islower = True
```

Comment:
This has the same bug as fixed here https://github.com/numba/numba/pull/4346

Reply:
@stuartarchibald True! I will make necessary changes and submit it again. Thanks for noticing that.